docs/sdk/v3/guides/liquidity/03-checking-collecting-fees.md

```diff
@@ -1,6 +1,57 @@
 ---
 id: liquidity-fees
-title: Checking & Collecting Fees
```

Comment:
Is there no way to check how much currency has been accrued from fees?

Reply:
No unfortunately